azurerm/import_arm_log_analytics_linked_service_test.go

```diff
@@ -0,0 +1,56 @@
+package azurerm
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAzureRMLogAnalyticsLinkedService_importRequiredOnly(t *testing.T) {
```

Comment:
We have been moving import tests to the end of the standard tests. Could we move this one there?

Thanks

Reply:
Did not know that. Moved it.